network/tls.go

```diff
@@ -306,7 +308,8 @@ type verifier func(rawCerts [][]byte, vrf [][]*x509.Certificate) (err error)
 // gives us a certificate back, crypto/tls calls the verifier with arguments we
 // can't control. But the verifier still has access to the nonce because it's in the
 // closure.
-func makeVerifier(suite Suite, them *ServerIdentity) (verifier, []byte) {
+func makeVerifier(suite Suite, them *ServerIdentity) (verifier,
```

Comment:
This is bad formatting. If this is the fault of the pre-commit stuff, it needs to be fixed somehow.